pkg/serve/instance_scope_internal_test.go

```
@@ -0,0 +1,240 @@
+	//: would otherwise compute the same hash but write to different
+	//: directories, each binding its own daemon for the same workspace.
+	//: Including RuntimeDir collapses that race onto the canonical path
+	//: each process sees.
+	if a.Hash() == b.Hash() {
+		t.Fatalf("Hash should partition by RuntimeDir, got identical %s", a.Hash())
+	}
+}
+
+func Test_instanceScopeOptions_Hash_SymlinkResolution(t *testing.T) {
+	tmp := t.TempDir()
+	real := filepath.Join(tmp, "real")
```

golangci-lint (2.12.1)

[error] variable real has same name as predeclared identifier

(predeclared)